lib/perl/Genome/Process/Command/Run.pm

```diff
@@ -148,7 +148,7 @@ sub _bsub_in_pend_state {
         log_file => $self->workflow_process_out_log,
         hold_job => 1,
         project => $self->process->workflow_name,
-        queue => $ENV{GENOME_LSF_QUEUE_BUILD_WORKFLOW},
+        queue => $ENV{GENOME_LSF_QUEUE_DV2_WORKFLOW},
```

Comment:
I believe that a `Build` that runs a `Process` should `execute` it rather than `schedule` it.